### ISSUE TITLE
Trying to compress with draco only logs `error: Error applying Draco compression.` for some meshes

### ISSUE THREAD

**Issue by hybridherbst** [donmccurdy/glTF-Transform#618]
**Describe the bug**
After merging some meshes in Blender, I can't run them through Draco compression anymore. Not sure if that is something that glTF-transform can help with or maybe even a Draco bug?

**To Reproduce**
1. Download [castle-trojan-horse.merged.zip](https://github.com/donmccurdy/glTF-Transform/files/9078024/castle-trojan-horse.merged.zip)
2. Run `gltf-transform draco castle-trojan-horse.merged.glb castle-trojan-horse.merged+draco.glb`
3. Note not very informative error logged

**Expected behavior**
Either compression works, or maybe a better error message that tells me something actionable

**Versions:**
 - Version: 2.1.8
 - Environment: CLI

**Comment by donmccurdy** [comment]
I don't think I know more than you do about the error — WebAssembly code does not tend to throw helpful errors — but if you include the `--method sequential` flag, compression does succeed. That has been true a few times in my experience, so I suspect it may be a Draco bug.

**Comment by donmccurdy** [comment]
Same error from gltf-pipeline...

```
➜ gltf-pipeline -i ~/Desktop/castle-trojan-horse.merged.glb -o ~/Desktop/castle-trojan-horse.merged+draco.glb -d -b
Error
    at new RuntimeError (/Users/donmccurdy/.nvm/versions/node/v14.18.1/lib/node_modules/gltf-pipeline/node_modules/cesium/Build/CesiumUnminified/Cesium.js:6891:13)
    at /Users/donmccurdy/.nvm/versions/node/v14.18.1/lib/node_modules/gltf-pipeline/lib/compressDracoMeshes.js:270:15
    at Function.ForEach.meshPrimitive (/Users/donmccurdy/.nvm/versions/node/v14.18.1/lib/node_modules/gltf-pipeline/lib/ForEach.js:239:21)
    at /Users/donmccurdy/.nvm/versions/node/v14.18.1/lib/node_modules/gltf-pipeline/lib/compressDracoMeshes.js:144:13
    at Function.ForEach.object (/Users/donmccurdy/.nvm/versions/node/v14.18.1/lib/node_modules/gltf-pipeline/lib/ForEach.js:44:21)
    at Function.ForEach.topLevel (/Users/donmccurdy/.nvm/versions/node/v14.18.1/lib/node_modules/gltf-pipeline/lib/ForEach.js:63:18)
    at Function.ForEach.mesh (/Users/donmccurdy/.nvm/versions/node/v14.18.1/lib/node_modules/gltf-pipeline/lib/ForEach.js:230:18)
    at compress (/Users/donmccurdy/.nvm/versions/node/v14.18.1/lib/node_modules/gltf-pipeline/lib/compressDracoMeshes.js:143:11)
    at /Users/donmccurdy/.nvm/versions/node/v14.18.1/lib/node_modules/gltf-pipeline/lib/compressDracoMeshes.js:60:14
    at tryCatcher (/Users/donmccurdy/.nvm/versions/node/v14.18.1/lib/node_modules/gltf-pipeline/node_modules/bluebird/js/release/util.js:16:23)
    at Promise._settlePromiseFromHandler (/Users/donmccurdy/.nvm/versions/node/v14.18.1/lib/node_modules/gltf-pipeline/node_modules/bluebird/js/release/promise.js:547:31)
    at Promise._settlePromise (/Users/donmccurdy/.nvm/versions/node/v14.18.1/lib/node_modules/gltf-pipeline/node_modules/bluebird/js/release/promise.js:604:18)
    at Promise._settlePromiseCtx (/Users/donmccurdy/.nvm/versions/node/v14.18.1/lib/node_modules/gltf-pipeline/node_modules/bluebird/js/release/promise.js:641:10)
    at _drainQueueStep (/Users/donmccurdy/.nvm/versions/node/v14.18.1/lib/node_modules/gltf-pipeline/node_modules/bluebird/js/release/async.js:97:12)
    at _drainQueue (/Users/donmccurdy/.nvm/versions/node/v14.18.1/lib/node_modules/gltf-pipeline/node_modules/bluebird/js/release/async.js:86:9)
    at Async._drainQueues (/Users/donmccurdy/.nvm/versions/node/v14.18.1/lib/node_modules/gltf-pipeline/node_modules/bluebird/js/release/async.js:102:5) {
  message: 'Error: Draco encoding failed.'
```

... so I have to assume this is an issue for the Draco library. Not sure how to make that more actionable, unfortunately.

**Comment by donmccurdy** [comment]
Attempting a fix in #981, with one open question in https://github.com/google/draco/issues/929.
